README.md

```diff
@@ -24,6 +24,100 @@ yarn install
 
 ```
 
+Ready your React .env:
+
+```bash
+
+cd packages/react-app
+cp .sample.env .env
+
+```
+
+Edit the new .env, & save changes, don't edit BACKEND_URL unless using custom backend.
+
+Next, ready your Hardhat .env:
+
+Navigating from /scaffold-directory..
+
+```bash
+
+cd packages/hardhat
+cp example.env .env
```

Comment:
We have `example.env`, `.env.sample` and `.sample.env` in the different packages :smiling_face_with_tear: 

I renamed all of them to `.env.sample` for consistency.